lib/getProxyFromURI.js

```diff
@@ -11,23 +11,23 @@ function formatHostname(hostname) {
 function parseNoProxyZone(zone) {
     zone = zone.trim().toLowerCase();
 
-    var zoneParts = zone.split(':', 2);
-    var zoneHost = formatHostname(zoneParts[0]);
-    var zonePort = zoneParts[1];
-    var hasPort = zone.indexOf(':') > -1;
+    let zoneParts = zone.split(':', 2);
+    let zoneHost = formatHostname(zoneParts[0]);
+    let zonePort = zoneParts[1];
+    let hasPort = zone.indexOf(':') > -1;
```

Comment:
These can all be `const`.

Also we don't need to a string operation to determine `hasPort`, we can just look at the length of the `zoneParts`:

```
const hasPort = zoneParts.length == 2;
```

Reply:
Done!